network/iptables.go

```diff
@@ -0,0 +1,9 @@
+package network
+
+type ipTablesClientInterface interface {
```

Comment:
simply call it `ipTablesClient`, we don't do [Hungarian notation](https://en.wikipedia.org/wiki/Hungarian_notation)